packages/blinks-core/src/api/BlinkConfig.ts

```diff
@@ -1,65 +1,65 @@
 import { Connection } from '@solana/web3.js';
 import {
-  type Action,
+  type BlinkInstance,
   AbstractActionComponent,
   DEFAULT_SUPPORTED_BLOCKCHAIN_IDS,
 } from './Action';
 import type { SignMessageData } from './actions-spec.ts';
 
-export interface ActionContext {
+export interface BlinkExecutionContext {
   originalUrl: string;
-  action: Action;
+  action: BlinkInstance;
   actionType: 'trusted' | 'malicious' | 'unknown';
   triggeredLinkedAction: AbstractActionComponent;
 }
 
-export interface IncomingActionConfig {
+export interface IncomingBlinkConfig {
   rpcUrl: string;
-  adapter: Pick<ActionAdapter, 'connect' | 'signTransaction' | 'signMessage'> &
-    Partial<Pick<ActionAdapter, 'metadata'>>;
+  adapter: Pick<BlinkAdapter, 'connect' | 'signTransaction' | 'signMessage'> &
+    Partial<Pick<BlinkAdapter, 'metadata'>>;
 }
 
 /**
- * Metadata for an action adapter.
+ * Metadata for an blink adapter.
  *
  * @property supportedBlockchainIds List of CAIP-2 blockchain IDs the adapter supports.
  *
  * @see {BlockchainIds}
  */
-export interface ActionAdapterMetadata {
+export interface BlinkAdapterMetadata {
   /**
    * List of CAIP-2 blockchain IDs the adapter supports.
    */
   supportedBlockchainIds: string[];
 }
 
-export interface ActionAdapter {
-  metadata: ActionAdapterMetadata;
-  connect: (context: ActionContext) => Promise<string | null>;
+export interface BlinkAdapter {
```

Comment:
also breaking change (leaving this note since phantom and other wallets can ask abt this)

Reply:
There is a renamed export at line 140, so no breaking

```ts
export { BlinkConfig as ActionConfig, type BlinkAdapter as ActionAdapter };
```